mllib/src/main/scala/org/apache/spark/ml/stat/KolmogorovSmirnovTest.scala

```diff
@@ -59,7 +59,7 @@ object KolmogorovSmirnovTest {
    * distribution of the sample data and the theoretical distribution we can provide a test for the
    * the null hypothesis that the sample data comes from that theoretical distribution.
    *
-   * @param dataset a `DataFrame` containing the sample of data to test
+   * @param dataset A dataset or a dataframe containing the sample of data to test
```

Comment:
nit: It's nicer to keep single back quotes ``` `DataFrame` ``` to make these show up as code in docs for clarity.  No need to get rid of that.